packages/core/src/types.ts

```diff
@@ -1904,6 +1932,26 @@ export type SnapshotFrom<T> = ReturnTypeOrValue<T> extends infer R
 export type EventFromBehavior<TBehavior extends ActorBehavior<any, any>> =
   TBehavior extends ActorBehavior<infer TEvent, infer _> ? TEvent : never;
 
+export type PersistedStateFrom<TBehavior extends ActorBehavior<any, any>> =
+  TBehavior extends ActorBehavior<
+    infer _TEvent,
+    infer _TSnapshot,
+    infer _TInternalState,
+    infer TPersisted
+  >
+    ? TPersisted
+    : 'nope';
```

Comment:
nope, this won't pass the review, nope

<img src="https://media.giphy.com/media/26hkhKd2Cp5WMWU1O/giphy.gif" alt="girl shaking her head and saying 'nope'"/>

Reply:
I need help with this 🥺

Reply:
all of the other types like this in our repo just use `never` here
```suggestion
    : never;
```